### PR TITLE
✨ Add first published at field to news #1038

### PR DESCRIPTION
**Pull request by fernandolucchesi**

### CODE REVIEW

File: .github/workflows/DEV-studio.yml [hunk 1]
```diff
@@ -11,11 +11,6 @@ jobs:
     runs-on: ubuntu-latest
     env:
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-      SANITY_STUDIO_BRANDMASTER_URL: ${{secrets.SANITY_STUDIO_BRANDMASTER_URL}}
```
Comment (SvSven) [inline .github/workflows/DEV-studio.yml:14]: Are these env variables no longer needed here or?

File: .github/workflows/TEST-studio.yaml [hunk 2]
```diff
@@ -6,11 +6,6 @@ jobs:
     runs-on: ubuntu-latest
     env:
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-      SANITY_STUDIO_BRANDMASTER_URL: ${{ secrets.SANITY_STUDIO_BRANDMASTER_URL }}
```
Comment (SvSven) [inline .github/workflows/TEST-studio.yaml:9]: Same as other question on env variables

File: studio/actions/ConfirmPublishWithi18n.ts [hunk 3]
```diff
@@ -130,10 +134,29 @@ export const ConfirmPublishWithi18nAction = ({ type, id, onComplete }: IResolver
...
+          method: 'GET',
+          headers: new Headers({
+            Authorization: `Bearer ${token}`,
+          }),
+        },
+      )
+        .then((res) => res.text())
+        .catch((err: Error) => console.error(err))
+
+      const hasBeenPublished = revisions?.includes(`"_id":"${document?._id.replace('drafts.', '')}"`)
```
Comment (SvSven) [inline studio/actions/ConfirmPublishWithi18n.ts:150]: Seems weird that this is the way to check if a document has been published before or not, but at the same time it doesn't surprise me.

**Review by SvSven — APPROVED** [review]
Couple of comments but it looks to be working on the studio end at least 👍

File: studio/actions/ConfirmPublishWithi18n.ts (hunk 3 above)
Comment (fernandolucchesi) [reply studio/actions/ConfirmPublishWithi18n.ts:150]: I couldn't find a better way to do this. I will ask in the sanity channel, but I do agree this is super ugly.

**Pull request merged**